_rules/SVG-image-has-name

```diff
@@ -0,0 +1,143 @@
+---
+name: SVG image has name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree has an accessible name.
```

Comment:
Do you think the ["image has accessible name" rule](https://auto-wcag.github.io/auto-wcag/rules/SC1-1-1-image-has-name.html) can be used to keep kind of the same wording/phrasing ? 

I suggest juste some few modifications:

> name : SVG image has *accessible* name
> 
> description : This rule checks that each SVG image element that is included in the accessibility tree, has an accessible name.